llvm/include/llvm/IR/Attributes.td

```diff
@@ -285,6 +285,9 @@ def SanitizeHWAddress : EnumAttr<"sanitize_hwaddress", [FnAttr]>;
 /// MemTagSanitizer is on.
 def SanitizeMemTag : EnumAttr<"sanitize_memtag", [FnAttr]>;
 
+/// NumericalStabilitySanitizer is on.
+def SanitizeNumericalStability : EnumAttr<"sanitize_numericalstability", [FnAttr]>;
```

Comment:
Not sure what is the rule here, why not `sanitize_numerical_stability` ?

Reply:
will do (already switched on https://github.com/llvm/llvm-project/pull/93783)

Reply:
Done